plugins/webp-uploads/hooks.php

```diff
@@ -803,3 +803,69 @@ function webp_uploads_enable_additional_mime_type_support_for_all_sizes( array $
 	return $allowed_sizes;
 }
 add_filter( 'webp_uploads_image_sizes_with_additional_mime_type_support', 'webp_uploads_enable_additional_mime_type_support_for_all_sizes' );
+
+/**
+ * Converts palette PNG images to true color PNG images.
```

Comment:
```suggestion
 * Converts palette PNG images to truecolor PNG images.
```

for whatever reason, PNG calls these "truecolor" - one word.